packages/charm/src/manager.ts

```diff
@@ -133,26 +147,48 @@ export class CharmManager {
   ) {
     this.space = this.session.space;
 
-    this.charms = this.runtime.getCell(
+    // Use the well-known ALL_CHARMS_ID entity for the charms cell
+    this.charms = this.runtime.getCellFromEntityId(
       this.space,
-      "charms",
+      { "/": ALL_CHARMS_ID },
+      [],
       charmListSchema,
     );
     this.pinnedCharms = this.runtime.getCell(
       this.space,
       "pinned-charms",
       charmListSchema,
     );
-    this.trashedCharms = this.runtime.getCell(
+    this.recentCharms = this.runtime.getCell(
```

Comment:
This seems wrong since we don't _need_ a fixed cause, but then what should the cause be...?